help/destinations/catalog/mobile-engagement/line.md

```diff
@@ -0,0 +1,141 @@
+![Platform UI screenshot showing the destination details.](../../assets/catalog/mobile-engagement/line/destination-details.png)
+
+* **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
+* **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+* **[!UICONTROL Audience Type]**: If the identities in Platform are of type *IDFA* select **[!UICONTROL IDFA]**. If the identities in Platform are of type *UserID* select **[!UICONTROL UserID]**. Refer to [Supported identities](#supported-identities) if you need guidance.
+
+### Enable alerts {#enable-alerts}
+
+You can enable alerts to receive notifications on the status of the dataflow to your destination. Select an alert from the list to subscribe to receive notifications on the status of your dataflow. For more information on alerts, see the guide on [subscribing to destinations alerts using the UI](../../ui/alerts.md).
+
+When you are finished providing details for your destination connection, select **[!UICONTROL Next]**.
+
+## Activate segments to this destination {#activate}
+
+>[!IMPORTANT]
+> 
+> To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+Read [Activate profiles and segments to streaming segment export destinations](../../ui/activate/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+### Map attributes and identities {#map}
+
+To correctly send your audience data from Adobe Experience Platform to the [!DNL LINE] destination, you need to go through the field mapping step. Mapping consists of creating a link between your Experience Data Model (XDM) schema fields in your Platform account and their corresponding equivalents from the target destination. To correctly map your XDM fields to the [!DNL LINE] destination fields, follow these steps:
+
+The following target identity namespace(s) must be mapped:
+* **UserID** must be mapped as a target field using **Target field** > **[!UICONTROL Select identity namespace]** > **LineId**
+
+* An example is shown below:
+![Platform UI screenshot example showing the Target mapping.](../../assets/catalog/mobile-engagement/line/mappings.png)
+
+## Validate data export {#exported-data}
+
+For each successful execution the [!DNL LINE] destination creates a new audience within [!DNL LINE] using the selected segments name.
+
+To validate that you have correctly set up the destination, follow the steps below:
+
+1. Log in to the [Manager console](https://manager.line.biz/).
```

Comment:
Minor, but to really set the context, let's update to: 

`In [!DNL LINE], log in to the [Manager console](https://manager.line.biz/).`